keanu-project/src/main/java/io/improbable/keanu/tensor/dbl/Nd4jDoubleTensor.java

```diff
@@ -833,6 +836,75 @@ public DoubleTensor concat(int dimension, DoubleTensor... those) {
         return new Nd4jDoubleTensor(concat);
     }
 
+    /**
+     * @param dimension      the dimension to split on
+     * @param splitAtIndices the indices that the dimension to split on should be split on
+     * @return pieces of the tensor split in the order specified by splitAtIndices. To get
+     * pieces that encompasses the entire tensor, the last index in the splitAtIndices must
+     * be the length of the dimension being split on.
+     * <p>
+     * e.g A =
+     * [
+     * 1, 2, 3, 4, 5, 6
+     * 7, 8, 9, 1, 2, 3
+     * ]
+     * <p>
+     * A.split(0, [1]) gives List([1, 2, 3, 4, 5, 6])
+     * A.split(0, [1, 2] gives List([1, 2, 3, 4, 5, 6], [7, 8, 9, 1, 2, 3]
```

Comment:
nit: missing closing bracket for split and List

Reply:
Nice eye.